apps/storybook/src/Alert.test.ts

```diff
@@ -0,0 +1,22 @@
+describe('Alert', () => {
+  const storyPath = 'Core/Alert';
+  const tests = [
```

Comment:
I was not able to make it work, maybe you know a better way. But even then, we would need to manually exclude the stories we don't want to test.

But I also didn't spend too much time on it because I wanted to avoid adding any storybook-specific logic (right now it's just urls) and we might move off storybook in the future.

Reply:
I had already tried that but couldn't get it to work. It tries to import and process the entire file, which doesn't work because it has css imports.

![Oops...we found an error preparing this test file: src/Alert.test.ts
The error was:
Error: Webpack Compilation Error
/e2e/node_modules/@itwin/itwinui-css/css/table.css 5:0
Module parse failed: Unexpected character '@' (5:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file.](https://user-images.githubusercontent.com/9084735/168854509-95d70d28-0a9f-4e24-9ab4-966a68de3753.png)